modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -116,21 +113,22 @@ func (k Keeper) storeWasmCode(ctx sdk.Context, code []byte) ([]byte, error) {
 
 	// create the code in the vm
 	ctx.GasMeter().ConsumeGas(types.VMGasRegister.CompileCosts(len(code)), "Compiling wasm bytecode")
-	codeHash, err := k.wasmVM.StoreCode(code)
+	vmCodeHash, err := k.wasmVM.StoreCode(code)
```

Comment:
what happens in case the bottom sanity check reveals that the two code hashes are not the same for some reason? i doubt this would ever happen, but i guess if it does then we've also already stored the code hash. not sure how much of a huge problem this is but i guess i would be confused by the error response if the codehash is stored anyway

Reply:
I think we talked about it and if tx gets reverted the storing of code will similarly get reverted. Will look into it again tho